deploy/templates/nstemplatetiers/base/cluster.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: base-cluster-resources
+objects:
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-compute
+  spec:
+    quota:
+      hard:
+        limits.cpu: 20000m
+        limits.memory: 7Gi
+        limits.ephemeral-storage: 7Gi
+        requests.cpu: 1750m
+        requests.memory: 7Gi
+        requests.storage: 15Gi
+        requests.ephemeral-storage: 7Gi
+        count/persistentvolumeclaims: "5"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-deployments
+  spec:
+    quota:
+      hard:
+        count/deployments.apps: "30"
+        count/deploymentconfigs.apps: "30"
+        count/pods: "50"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-replicas
+  spec:
+    quota:
+      hard:
+        count/replicasets.apps: "30"
+        count/replicationcontrollers: "30"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-routes
+  spec:
+    quota:
+      hard:
+        count/ingresses.extensions: "10"
+        count/routes.route.openshift.io: "10"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-jobs
+  spec:
+    quota:
+      hard:
+        count/jobs.batch: "30"
+        count/daemonsets.apps: "30"
+        count/cronjobs.batch: "30"
+        count/statefulsets.apps: "30"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-services
+  spec:
+    quota:
+      hard:
+        count/services: "10"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}-bc
+  spec:
+    quota:
+      hard:
+        count/buildconfigs.build.openshift.io: "30"
```

Comment:
I wonder if we need a specific ClusterResourceQuota for each kind of resource? (secrets, buildconfigs, configmaps, etc.). Could we use a single cluster quota resource to specify all the counts at once instead?

Reply:
We had to split the ClusterResourceQuotas because of issue https://issues.redhat.com/browse/CRT-941. (https://github.com/codeready-toolchain/host-operator/pull/375)